plugin/install.go

```diff
@@ -131,6 +168,10 @@ func setupPluginDir(pluginDir string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	err = os.MkdirAll(filepath.Join(pluginDir, "meta"), 0755)
+	if err != nil {
+		return "", err
+	}
```

Comment:
Add tests to `TestSetupPluginDir`, please.

Reply:
fixed in df6507cea4317db45bcd62ce9c12c49765a8096b